src/Stratis.Features.FederatedPeg/Wallet/FederationWalletManager.cs

```diff
@@ -148,6 +153,91 @@ public class FederationWalletManager : LockProtected, IFederationWalletManager
             this.federatedPegSettings = federatedPegSettings;
             this.withdrawalExtractor = withdrawalExtractor;
             this.isFederationActive = false;
+            this.blockStore = blockStore;
+        }
+
+        private Dictionary<TransactionData, Transaction> GetSpendingTransactions(IEnumerable<TransactionData> transactions)
```

Comment:
Can you add a summary describing what this method does?

Reply:
Done.